md/migration-tool.md

```diff
@@ -5,6 +5,15 @@ to the one delivered with the latest version of Bonita.
 This is due to the fact that improvements in any version of the Migration Tool can affect all supported version of Bonita.
 This change log must be read before [migrating to a newer version of Bonita](migrate-from-an-earlier-version-of-bonita-bpm.md).
 
+## 2.40.0 - Jun. 6th, 2019 (Bonita 7.9.0)
```

Comment:
❓ I suggest to put a dedicated section on top of the page.

When a future version of the tool will be released, this will still apply for the 7.9.0 migration step and putting this information only on the 2.40.0 version may not emphasize this warning enough